src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php

```diff
@@ -37,7 +37,7 @@
  *      }
  * )
  */
-final class RaSecondFactor implements JsonSerializable
+class RaSecondFactor implements JsonSerializable
```

Comment:
While we'll probably never use 'em, final classes can mess up Doctrine's proxies (it's all in the commit messages ;) )